Homeworks/HW_#3_Class/app3.js

```diff
@@ -0,0 +1,55 @@
+class Vector {
+    constructor(vectorArray) {
+        this.vectorArray = vectorArray;
+    }
+
+    #getValidVector(inputtedVector) {
+        if (inputtedVector.vectorArray.length === this.vectorArray.length) {
+            return true
+        }
+        throw "Error"
```

Comment:
I suggest we choose one of the two approaches.
Either name function getIsValidVector and make it return true when the vector is valid and false when the function is not valid.
Another approach we can name function validateVector and throw an error when vector is not valid.

It's not good that we've mixed both ideas.